contributing/code/standards.rst

```diff
@@ -160,6 +160,9 @@ Structure
   ``tearDown`` methods of PHPUnit tests, which should always be the first methods
   to increase readability;
 
+* Declare all the arguments in the same line as the method/function name, no
```

Comment:
See the second example of http://www.php-fig.org/psr/psr-2/#4-1-extends-and-implements